frontends/mit-open/src/routes.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { RouteObject, Outlet } from "react-router"
+import HomePage from "./pages/Home"
+import LearningPathListingPage from "./pages/learningpaths/LearningPathListingPage"
+import LearningPathDetailsPage from "./pages/learningpaths/LearningPathDetails"
+import ArticleDetailsPage from "./pages/articles/ArticleDetailsPage"
+import ArticlesEditPage from "./pages/articles/ArticlesEditPage"
+import ErrorPage from "./pages/errors/ErrorPage"
+import * as urls from "./pages/urls"
+import * as deprecatedUrls from "./infinite-pages/urls"
+import EditFieldPage from "./infinite-pages/field-details/EditFieldPage"
+import FieldPage from "./infinite-pages/field-details/FieldPage"
+
+import Header from "./components/Header"
+
+const routes: RouteObject[] = [
+  {
+    element: (
+      <>
+        <Header />
+        <Outlet />
+      </>
+    ),
```

Comment:
RouteObjects are `{ path?, element?, children?, errorElement? }`.

Our top-level route is a "layout route" with no `path`. This means that (a) it is always rendered, and (b) child routes are rendered into the `<Outlet />`.